content/articles/statefull-vs-stateless-architectures/index.md

```diff
@@ -0,0 +1,116 @@
+
+### Learning Points 
+The article will cover:
+-	How stateful and stateless architectures affect the running of an application/ service on a server. 
+-	A short history to get you acquainted with what stateful and stateless application/ services mean. 
+-	The various protocols utilize each of the listed architectures. 
+-	The difference between the architectures 
+-	The advantages and disadvantages of the two architectures. 
+Let's get started!
+
+### Prerequisite
+1.	You should know internet protocols such as HTTP, UDP, TCP DNS SMTP, FTP, TCP, POP3 and TELNET as I will only feature a basic discussion of these protocols within the article.
+2.	Have a basic understanding of the difference between Graph QL, REST and SOAP application programming interfaces (APIs) 
+3.	You should have founding knowledge of networks bandwidths, servers, and clients.
+
+### Introduction 
+During the 1980s,  servers handled most the complex computations as clients were limited in their scope of operations that they could fulfill . However, as time passed, a global increase in web usage led to the development of the Hypertext Transfer Protocol (HTTP), web browsers, and other client-side software and hardware.
+
+Coincidentally, the rise in web usage led to an increase in the number of clients, as the clients could also handle many of the functions previously performed on servers.
+
+### What Are Stateful And Stateless Architectures 
+
+These are the architectural frameworks that define how users interact with services on the internet. Stateful architecture-based services maintain track of clients records or sessions and apply different functionalities to similar user inputs based on the user's previous records. In a stateless design, services rely on the clients to keep track of transaction records, allowing them to concentrate on operations that change the system’s resources rather than its state.
+
+The main difference between the two architectures is how they handle states. In a stateful based system, additional information is reserved on the server-side, saving the present transaction's state as it waits for the proceeding user requests. 
+
+In a stateless build system, extra information is  held on the client-side. Thus, as the system conveys different information, each session'reminds' the server of its preceeding sessions through the use of Cookies.
+
+### Network Protocols and their Architectures
+
+Network protocols are divided into two: 
+- Stateful protocols 
+- Stateless protocols. 
+
+### Examples
```

Comment:
Restructure these subheadings to:  Examples of Stateless Protocols 
                                                          Examples of Stateful protocols